nucliadb_utils/nucliadb_utils/storages/gcs.py

```diff
@@ -226,7 +226,14 @@ async def start(self, cf: CloudFile) -> CloudFile:
             quote_plus(upload_uri),
         )
         metadata = json.dumps(
-            {"FILENAME": cf.filename, "SIZE": cf.size, "CONTENT_TYPE": cf.content_type}
+            {
+                "contentType": cf.content_type,
```

Comment:
Content type should be duplicated?

Reply:
I left it duplicated for bw/compatibility, just in case.

`contentType` is part of the contract with GCS, whereas `metadata.CONTENT_TYPE` is part of the custom metadata we control

Reply:
actually, that is inferred from the header, so you are right, we don't need to duplicate it